config/environments/development.rb

```diff
@@ -62,4 +62,15 @@
 
   # add gem devise
   config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+
+  config.action_mailer.delivery_method = :smtp
+  # SMTP settings for gmail
+  config.action_mailer.smtp_settings = {
+    :address => "smtp.gmail.com",
+    :port => 587,
+    :user_name => ENV["USER_EMAIL"],
+    :password => ENV["USER_PASSWORD"],
```

Comment:
thêm env, thế cái env-sample em không thêm vào à

Reply:
em phải thêm vào cả env-sample để sau ngta lấy code máy em còn b có những biến môi trường nào nữa

Reply:
hehe.e thêm từ lúc đầu trong file .env-example rồi a :)))

Reply:
tính toán cả rồi a.haha